crates/ruff_dev/src/generate_rules_table.rs

```diff
@@ -22,7 +22,9 @@ fn generate_table(table_out: &mut String, rules: impl IntoIterator<Item = Rule>,
 
         #[allow(clippy::or_fun_call)]
         table_out.push_str(&format!(
-            "| {}{} | {} | {} | {} |",
+            "| <div id='{}{}'>{}{}</div> | {} | {} | {} |",
```

Comment:
```suggestion
            "| <div id='{0}{1}'>{0}{1}</div> | {2} | {3} | {4} |",
```
You can remove the next lines by using this.

Reply:
Thanks! Updated in 50b70c686cceae67c61c6fca93e6fa2fe78c5c67